docs/index.md

```diff
@@ -14,6 +14,14 @@ The current releases have been tested on minikube v1.12.3 running k8s v1.18.3
 - NXRM: `helm install nexus-repo sonatype/nexus-repository-manager`
 - Nexus IQ: `helm install nexus-iq sonatype/nexus-iq-server`
 
+### Get the Values for Configuring a Chart
+
+- NXRM: `helm show values nexus-repo sonatype/nexus-repository-manager`
+- Nexus IQ: `helm show values nexus-iq sonatype/nexus-iq-server`
+
+Capture that output as your own `values.yaml` file, and provide it to the `helm install` 
+command with the `-f` option.
```

Comment:
should the example `helm install` command immediately above have the `-f` option as an illustration? I'm okay as-is, but it looks like we're giving examples of all commands.

Reply:
i was debating that. it should install fine with defaults, so i didn't want to necessarily imply that it was always required. How much do I expect the helm user to know the command-line? Should they know to add the -f or know they can remove it?